meta-lmp-bsp/dynamic-layers/stm-st-stm32mp/recipes-bsp/trusted-firmware-a/tf-a-tools_2.7.bb

```diff
@@ -4,10 +4,9 @@ LIC_FILES_CHKSUM = "file://license.rst;md5=1dd070c98a281d18d9eefd938729b031"
 
 SRC_URI = "git://github.com/ARM-software/arm-trusted-firmware.git;protocol=https;branch=master \
            file://0001-tools-allow-to-use-a-root-key-password-from-command-.patch \
-           file://0002-fix-fiptool-respect-OPENSSL_DIR.patch \
```

Comment:
Maybe it's good to mention why this patch is being removed (is it upstream already?)

Reply:
These is a backported patch https://github.com/foundriesio/meta-lmp/pull/792/commits/5866016e47477e689ac9a10fa0e7fafdb469ed06#diff-b5e33ff710ac85f7839b8785da65e0f6063017c1cb7ae0f1b81720c8e9cefd83R12 from upstream version 2.7

```
Upstream-Status: Backport [https://github.com/ARM-software/arm-trusted-firmware/commit/dd14d0f63f7d049cbcbfd3f8128f05e9a351bbe5]
```